airflow/cli/cli_parser.py

```diff
@@ -626,6 +626,16 @@ def positive_int(value):
     ),
     action="store_true",
     default=False)
+ARG_CONN_EXPORT_FORMAT = Arg(
+    ('--format',),
+    help='Output format of the file for exporting the connections',
+    default='json',
+    type=str)
+ARG_CONN_EXPORT_FILE = Arg(
+    ('--output-file',),
```

Comment:
```suggestion
    ('output-file',),
```
What do you think about using positional arguments?
Before:
```
airflow connections export --output-file a.json
```
Before:
```
airflow connections export a.json
```